src/experiments.ts

```diff
@@ -164,6 +164,33 @@ export function setEnabled(name: ExperimentName, to: boolean | null): void {
   }
 }
 
+/**
+ * Returns the unsaved set of preferences.
+ */
+export function getSnapshotOfExperimentPreferences() {
+  return {
+    ...localPreferences(),
+  };
+}
```

Comment:
I don't love having to add this just for testing. Could `isEnabled` be used to check? That's how the real code would do it and it's probably wise to match that anyway.

Reply:
It could! Removed `getSnapshot` and am using `isEnabled` instead.